parser/tools/generate_ast.cc

```diff
@@ -208,7 +208,7 @@ NodeDef nodes[] = {
     {
         "EncodingLiteral",
         "__ENCODING__",
-        vector<FieldDef>(),
+        vector<FieldDef>({{"name", FieldType::Name}}),
```

Comment:
What's the point of this additional `name` field? It seems like it will always be equal to `__ENCODING__`

Reply:
You're right, it's not needed here, I removed it
@jez